xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -193,93 +193,19 @@ public boolean equals(Object o) {
   }
 
   static final class CdsUpdate implements ResourceUpdate {
```

Comment:
TODO in a separate PR: I may want to clean up other resource update data structures as well, they are too verbose to use.